components/Form/Select/Select.module.scss

```diff
@@ -0,0 +1,8 @@
+@import 'lbh-frontend/lbh/base';
+
+.inputContainer {
+  position: relative;
+  display: flex;
+  max-width: 48ex;
```

Comment:
i've not come across an `ex` unit before? should it be `px`?

Reply:
`px` would be too small of a max width and would cause the select's to be tiny, haven't used `ex` myself but it has been used elsewhere such as in the autocomplete